di_website/templates/vacancies/vacancy_index_page.html

```diff
@@ -39,17 +39,22 @@
     </div>
   </section>
 
-  {% if vacancies|length %}
-    <section class="section">
-      <div class="row row--narrow">
-        <div class="l-2up-3up">
+
+  <section class="section">
+    <div class="row row--narrow">
+      <div class="l-2up-3up">
+        {% if vacancies|length %}
           {% for vacancy in vacancies %}
             {% include 'includes/cards/card-job.html' %}
           {% endfor %}
-        </div>
+        {% else %}
+          <div class="is-typeset max-meter">
+            <p>No vacancies available</p>
```

Comment:
We should ask comms for guidance on what you'd like for a no vacancy message. I would think something like "Sorry, we have no vacancy postings at the moment. Please subscribe to the mailing list below to be notified as soon as new vacancies are posted."

Reply:
@akmiller01 comms provided guidance and have updated the text